ngraph/core/include/ngraph/op/util/gather_nd_base.hpp

```diff
@@ -0,0 +1,16 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "ngraph/op/op.hpp"
+#include "openvino/op/util/gather_nd_base.hpp"
+
+namespace ngraph {
+namespace op {
+namespace util {
+using ov::op::util::GatherNDBase;
```

Comment:
I don't think that we need to introduce headers with aliases to some new utility operations which are not included to official OpenVINO opsets.

Reply:
I used same header as for GatherBase operation. Do I need to remode this intoduction?